contracts/HATToken.sol

```diff
@@ -485,7 +485,7 @@ contract HATToken is IERC20 {
 
     function add96(uint96 a, uint96 b, string memory errorMessage) internal pure returns (uint96) {
         uint96 c = a + b;
-        require(c >= a, errorMessage);
+        assert(c >= a);
```

Comment:
We have a linter warning `errorMessage` is unused - can you remove the argument?